e2e/native-esm/__tests__/native-esm.test.js

```diff
@@ -28,13 +28,34 @@ test('should have correct import.meta', () => {
   expect(typeof require).toBe('undefined');
   expect(typeof jest).toBe('undefined');
   expect(import.meta).toEqual({
+    dirname: expect.any(String),
+    filename: expect.any(String),
     jest: expect.anything(),
     url: expect.any(String),
   });
   expect(import.meta.jest).toBe(jestObject);
   expect(
     import.meta.url.endsWith('/e2e/native-esm/__tests__/native-esm.test.js'),
   ).toBe(true);
+  if (process.platform === 'win32') {
```

Comment:
is this correct? If it's a `file://` thing, it should use forward slashes. See examples in https://nodejs.org/api/url.html#urlpathtofileurlpath

Reply:
I don't have access to a windows PC, or this'd be easy to check 😅 

@G-Rath would you be able to quickly check for us what `import.meta.filename` and `import.meta.dirname` returns on a windows machine?

Reply:
@SimenB filename and dirname are OS specific, but meta.url has URL format with forward slashes

Reply:
Perfect, thanks @G-Rath!